packages/support/lib/image-util.js

```diff
@@ -44,23 +38,22 @@ const MATCH_NEIGHBOUR_THRESHOLD = 10;
 
 const AVAILABLE_DETECTORS = [
   'AKAZE',
-  'AGAST',
+  'AgastFeatureDetector',
   'BRISK',
-  'FAST',
-  'GFTT',
+  'FastFeatureDetector',
```

Comment:
Can we just rewrite alises in the client code to not break the backward compatibility (e.g. AGAST -> AgastFeatureDetector) instead of forcing clients to change their code?

Reply:
yeah ok good idea